cli/chaosli/cmd/root.go

```diff
@@ -23,6 +22,7 @@ import (
 // Version will be set with the -ldflags option at compile time
 var Version = "v0"
 var cfgFile string
+var client ddmark.Client
```

Comment:
Same as above, I feel like it should be renamed ddmarkClient to be more explicit